squirrel.frontier/src/test/java/org/dice_research/squirrel/predictor/impl/PredictorImplTest.java

```diff
@@ -0,0 +1,181 @@
+
+        // Model and Classifier set up
+        predictor.model = new RegressionModel(weights,  new SigmoidActivationFunction());
+        RegressionClassifier classifier = new RegressionClassifier(predictor.model);
+
+        // Generate feature vector
+        predictor.featureHashing(curi);
+
+        // Prediction
+        double pred = predictor.predict(curi);
+
+        Assert.assertEquals(0.5587385795221708, pred, 1e-4);
```

Comment:
Again, I do not really believe that you have calculated the expected value by yourself :thinking: